pkg/utils/common.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// LastWorkUpdateTimeAnnotationKey is used to mark the last update time on a work object.
 	LastWorkUpdateTimeAnnotationKey = "work.fleet.azure.com/last-update-time"
+
+	// ResourceIdentifierStringFormat is the format of the resource identifier string.
+	ResourceIdentifierStringFormat = "%s/%s/%s/%s/%s"
```

Comment:
can be moved to the test util package? I don't think we'll use it in the production code.